README.md

```diff
@@ -14,10 +14,15 @@ Focused on separating concerns of development and operation needs, _Open Applica
 
 ## Introduction
 
+The motivation behind Open Application Model is to provide a standard but flexible framework for platform builders to create application centric platforms based on lower level orchestration systems like Kubernetes, in approach with discoverability, manageability and interoperability. 
+
 ![How it works][how-it-works]
 
-When it comes to application development and deployment, we think it is important to distinguish between the parts that developers are responsible for, and the parts that operations is responsible for. Otherwise, if these roles get muddled, it would result in communications mishaps, bugs, or even
-service outages.
+Open Application Model enables the platforms expose standardized application centric primitives (e.g. Workloads and Traits) instead of leaking infrastructure details to end user, and at the same time, keeps the extensibility of the underlying orchestration system. Though with the idea of making creating upper layer platforms easier, the model itself won't lock you in arbitrary abstractions, on the contrary, its primitives allow you to define the right level of abstractions by yourself.
+
+### A team-centric model
+
+When it comes to the application centric primitives, we think it is important to distinguish between the parts that developers are responsible for, and the parts that operations is responsible for. Otherwise, if these roles get muddled, it would result in communications mishaps, bugs, or even service outages.
```

Comment:
```suggestion
When it comes to the application centric primitives, we think it is important to distinguish between the parts that developers are responsible for, and the parts that operations are responsible for. Otherwise, if these roles get muddled, it would result in communications mishaps, bugs, or even service outages.
```
And `operations` here means `infrastructure operators` and `application operators`?

Reply:
It should be a typo